sample-app-android/README.md

```diff
@@ -33,3 +33,48 @@ Observable.fromCompletionStage(client.getItems(Article.class))
           // ...
         });
 ```
+
+### Using Appetize
+
+The application is preset to be able to consume data from different project (with the same scheme as in sample project). This feature is used for showcasing the application using [Appetize](https://appetize.io) service.
+
+To be able to consume the external configuration, the app checks whether the [Appetize Playback option - params](https://docs.appetize.io/core-features/playback-options) where provided and if so, it will reinitialize the client with specific project ID (and preview API key if provided).
+
+The decision logic is stored in `BaseActivity.java#onCreate` method.
+
+If you want to then use the Appetize you can create an Appetize iframe and provide `KontentProjectId` (and `KontentPreviewApiKey` if you what to use unpublished content) like this:

```

Comment:
I would remove "then"
Also where should I put this? I tested it through the website so probably you can also mention that it is the other option and manually add the KontentProjectId parameters in the URL.